lablup/backend.ai-webui
# add: confirmation dialog on force-terminate attempt on PULLING/PREPARING/TERMINATING

* Related: [backend.ai#2275](https://github.com/lablup/backend.ai/pull/2275)
This PR adds confirmation dialog when the user wants to "force-terminate" session(s).

### Screenshot(s):
* Superadmin trying to terminate session(s) which contains actual containers:
<img width="379" alt="Screenshot 2024-07-03 at 8 06 44 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/6795e212-6f85-4b57-b4db-73bd5517fcd5">

* User trying to terminate session(s) regardless of the session actually have containers or Superadmin trying to terminate session(s) which does not contain any containers:
<img width="379" alt="Screenshot 2024-07-03 at 8 06 59 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/5dbb6914-29a2-4aef-85df-3a584ec83aae">

### How to test
1. Go to sessions page.
2. Click Terminate icon or check any session records in the session list.
3. Terminate session dialog pops up.
4. Click `FORCE TERMINATE` button.
5. Force terminate confirmation dialog pops upl.
6. Click `FORCE TERMINATE` button in the dialog.
7. See the result. (session(s) in `PREPARING`/`PULLING`/`TERMINATING` will be updated to `TERMINATED` accordingly.)
   - When you logged in with user account or just admin account, the force-terminate operation will fail.
   - However, When you logged in with superadmin account, it will be successfully done.

**Checklist:** (if applicable)
